deepsensor/model/model.py

```diff
@@ -1084,13 +1084,47 @@ def stitch_clipped_predictions(
                         )
 
                         patches_clipped[var_name].append(patch_clip)
-
-            combined = {
-                var_name: xr.combine_by_coords(patches, compat="no_conflicts")
-                for var_name, patches in patches_clipped.items()
-            }
-
-            return combined
+            
+            # Create blank prediction dataframe. 
+            patchwise_pred_copy = copy.deepcopy(patches_clipped)
+
+            # Generate new blank DeepSensor.prediction object with same extent and coordinate system as X_t.
+            for var_name_copy, data_array_list in patchwise_pred_copy.items():
+                first_patchwise_pred = data_array_list[0]
+
+                # Define coordinate extent and time
+                blank_pred_copy = xr.Dataset(coords={orig_x1_name: X_t[orig_x1_name], 
+                                                    orig_x2_name: X_t[orig_x2_name],
+                                                    'time': first_patchwise_pred['time']}) # Is this fine or can 'time' assume a different name?'
+
+                # Set variable names to those in patched predictions, set values to Nan. 
+                for var_name_i in first_patchwise_pred.data_vars:
+                    blank_pred_copy[var_name_i] = first_patchwise_pred[var_name_i]
+                    blank_pred_copy[var_name_i][:] = np.nan
+                patchwise_pred_copy[var_name_copy]= blank_pred_copy
+
+            # Merge patchwise predictions to create final combined dataset. 
+            combined_dataset = patchwise_pred_copy  # Use the previously initialized dictionary
```

Comment:
Thanks, I understand the reasoning. I'll look into refactoring this a little since creating another copy of a variable just for it to have a different name is considered bad practice (as it uses more memory than is necessary) but it indicates to me that tweaking the logic here might be required.